Lekce_05_BR01/Program.cs

```diff
@@ -81,7 +78,7 @@ public static void Main()
         };
 
             // 4. Řešení
-            List<string> vzestupne = new List<string>();
+            List<string> vzestupne = jmena.OrderBy(x => x).ToList();
```

Comment:
nit: V .NET 7 pribyla nova metoda co to zjednodusuje, pokud chces radit pomoci samotneho elementu :), takze se to da zjednodusit takhle
```suggestion
            List<string> vzestupne = jmena.Order().ToList();
```